### DISCUSSION TITLE
SE Project Discussion

### DISCUSSION THREAD

**Discussion by Fashmina123** [abirChakrabortyIITM/seproject#33] [General]
This is the first comment on this discussion. Post your comments and queries here!

**Comment by abirChakrabortyIITM** [comment]
I have pulled the recent changes and the Delete button does not work for me in the frontend, i request the frontend team to please look into it.

**Reply by raj-ranjan-01** [reply]
let me check the code and get back to you

**Reply by abirChakrabortyIITM** [reply]
sure Raj! thank you
